src/main/java/frc/robot/commands/ShooterBaseCommand.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.commands;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.shooter.ShooterBase;
+import java.util.Map;
+import java.util.function.BooleanSupplier;
+
+public class ShooterBaseCommand extends Command {
+  ShooterBase shooter;
+  BooleanSupplier enable;
+  ShuffleboardTab tab;
+  GenericEntry voltsEntry;
+
+  public ShooterBaseCommand(ShooterBase shooter, BooleanSupplier enable) {
```

Comment:
Is this ever going to be called to _disable_ the shooter?  If not, this can just be a "ShooterEnable(ShooterBase shooter)"

Reply:
I don't think so. I will rename the Command.